src/main/java/com/raisetech/mybatisdemo/repository/UserMapper.java

```diff
@@ -0,0 +1,17 @@
+package com.raisetech.mybatisdemo.repository;
+
+import com.raisetech.mybatisdemo.entity.User;
+import org.apache.ibatis.annotations.Insert;
+import org.apache.ibatis.annotations.Mapper;
+import org.apache.ibatis.annotations.Select;
+
+import java.util.List;
+
+@Mapper
+public interface UserMapper {
+    @Select("SELECT * FROM users")
+    List<User> findAll();
+
+    @Insert("insert into users (name,address,age) values (#{name},#{address},#{age})")
```

Comment:
```suggestion
    @Select("SELECT * FROM users")
    List<User> findAll();

    @Insert("INSERT INTO users (name, address, age) VALUES (#{name}, #{address}, #{age})")
```
MySQLの予約語は大文字に揃えたいですね。
全部小文字に統一するでもよいですが。
それからカンマの後には半角スペースを入れたいです。

Reply:
> MySQLの予約語は大文字に揃えたいですね。

SQLの書き方を全然わかっていませんでした。修正しました！